### PR TITLE
Mailing list notifier

### PR DESCRIPTION
**Pull request by lukasbischof**

### CODE REVIEW

File: app/mailers/mailing_list_entry_notifier_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class MailingListEntryNotifierMailer < ApplicationMailer
+  TO = 'joshua.dedevadas@gmail.com'
+  CC = %w[me@luk4s.dev philipp@thefehr.me].freeze
+  FROM = "MyZivi Mailing-List Notifier <#{ENV['MAIL_SENDER']}>"
```
Comment (TheFehr) [inline app/mailers/mailing_list_entry_notifier_mailer.rb:6]: Shouldn't this be a env? I mean now our email addresses are public on github?

**Review by TheFehr — CHANGES_REQUESTED** [review]

File: app/mailers/mailing_list_entry_notifier_mailer.rb (hunk 1 above)
Comment (lukasbischof) [reply app/mailers/mailing_list_entry_notifier_mailer.rb:6]: Well, they're anyway public through commits 😂

Comment (TheFehr) [reply app/mailers/mailing_list_entry_notifier_mailer.rb:6]: Yeah, but...

Comment (lukasbischof) [reply app/mailers/mailing_list_entry_notifier_mailer.rb:6]: Yes?

**Review by TheFehr — APPROVED** [review]

**Pull request merged**